src/Types/AbstractTypedIDEntity.php

```diff
@@ -111,7 +104,7 @@ public function toArray(): array
      */
     public function toJson($options = 0): string
     {
-        return (string) \json_encode($this->toArray(), $options);
+        return Json::encode((object) $this->toArray(), $options);
```

Comment:
> cast to `object`

What is this for?

Reply:
In json context this is not required, you right. FIxed